testing/test_vulkan_context.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TESTING_TEST_VULKAN_CONTEXT_H_
+#define FLUTTER_TESTING_TEST_VULKAN_CONTEXT_H_
+
+#include "flutter/vulkan/vulkan_application.h"
+#include "flutter/vulkan/vulkan_device.h"
+#include "flutter/vulkan/vulkan_proc_table.h"
+
+namespace flutter {
+
+/// @brief  Utility class to create a Vulkan device context, a corresponding
+///         Skia context, and device resources.
+class TestVulkanContext {
+ public:
+  TestVulkanContext();
+  ~TestVulkanContext();
+  bool is_valid();
+
+ private:
+  bool valid_;
+  fml::RefPtr<vulkan::VulkanProcTable> vk_;
+  std::unique_ptr<vulkan::VulkanApplication> application_;
+  std::unique_ptr<vulkan::VulkanDevice> logical_device_;
+};
```

Comment:
`FML_DISALLOW_COPY_AND_ASSIGN` explicitly even though this can't really be copied because of the unique pointers.

Reply:
Done.